app/assets/javascripts/map/cartocss/style.cartocss

```diff
@@ -61,6 +61,7 @@
   [layer='glad_coverage'],
   [layer='geographic_coverage_pantropical'],
   [layer='umd_coverage_2015'] {
```

Comment:
we need to be careful with the code reviews guys. This broke Production. Fixing. (see the extra `{`)